orcid-core/src/main/java/org/orcid/core/manager/read_only/impl/AffiliationsManagerReadOnlyImpl.java

```diff
@@ -124,8 +123,7 @@ public EmploymentSummary getEmploymentSummary(String userOrcid, Long employmentI
      * @return the list of employments that belongs to this user
      * */
     @Override
-    @Cacheable(value = "employments-summaries", key = "#userOrcid.concat('-').concat(#lastModified)")
```

Comment:
If both managers cache the same thing we end up with objects from different versions in the same cache, causing class cast exceptions